server/routes/utils/reportHelper.ts

```diff
@@ -53,7 +56,7 @@ export const generatePNG = async (
     //TODO: Add header and footer, phase 2
 
     await browser.close();
-    return { timeCreated, stream, fileName };
+    return { timeCreated, stream: buffer.toString('base64'), fileName };
```

Comment:
There is a better way to do base64 encoding, right from puppeteer.

Reply:
Will check